### PR TITLE
adding dictdiffer recipe

### PR DESCRIPTION
**Pull request by hodgsonw**
dictdiffer is a python tool for diffing, and patching dictionaries

### CODE REVIEW

File: recipes/dictdiffer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - dictdiffer
+  requires:
+    - coverage >=4.0
+    - isort >=4.2.2
+    - mock >=1.3.0
+    - pydocstyle >=1.0.0
+    - pytest >=2.8.0
+    - pytest-cache >=1.0
+    - pytest-cov >=1.8.0
+    - pytest-pep8 >=1.0.6
```
Comment (isuruf) [inline recipes/dictdiffer/meta.yaml]: Since the test is just importing the package, you can remove the test requirements above

Comment (kmuehlbauer) [reply recipes/dictdiffer/meta.yaml]: @isuruf Remove the `require` section at all?

**Comment by kmuehlbauer** [comment]
@hodgsonw Could you take over from here? I somehow can't PR the changes to your branch. And please add me to maintainers.

**Comment by kmuehlbauer** [comment]
@hodgsonw We might need *setuptools* somewhere. Do you have any idea, why this is breaking in test?

**Comment by kmuehlbauer** [comment]
@conda-forge/staged-recipes This can be closed, recipe is merged here #7168

**Comment by ocefpaf** [comment]
> @conda-forge/staged-recipes This can be closed, recipe is merged here #7168

B/c you picked the commits from this PR it was automatically closed (actually merged) when we merge #7168.

**Comment by kmuehlbauer** [comment]
@ocefpaf Very nice! Didn't know that. So it's actually a very neat way.

**Pull request merged**